ui/pages/remote-mode/overview/remote-mode-overview.container.tsx

```diff
@@ -48,6 +49,7 @@ export default function RemoteModeIntroducing() {
   const [currentScreen, setCurrentScreen] = useState<RemoteScreen>(
     RemoteScreen.OVERVIEW,
   );
+  const [isInitialized, setIsInitialized] = useState<boolean>(false);
   const [isHardwareAccount, setIsHardwareAccount] = useState<boolean>(false);
```

Comment:
If you set the default state for `isHardwareAccount` to `true` (assume you start with a valid account), would that achieve the same behavior, without the extra piece of state?

Reply:
Doh, yeah good call :) Reverted in 8ac8615ebc329f22ee46630b0ef9a45fe858f9f3